gammapy/data/observations.py

```diff
@@ -359,6 +382,31 @@ def check_observation(self):
 
         return messages
 
+    def to_observation_cta(self):
+        """
+        Convert a `~gammapy.data.DataStoreObservation` to an `~gammapy.data.ObservationCTA`
+
+        Returns
+        -------
+        obs : `~gammapy.data.ObservationCTA`
+            An observation container that stores all class attributes in memory
+
+        """
+        # maps the ObservationCTA class attributes to the DataStoreObservation properties
+        obs_cta_kwargs = {'obs_id': 'obs_id', 'gti': 'gti', 'events': 'events', 'aeff': 'aeff', 'edisp': 'edisp',
```

Comment:
Looks like you have a dict where key val are always the same. Use a list instead, no?

Reply:
Yes, i started with a list, but then i had to create a kwargs dict for the return in the end, and i thought like this it would be nicer. It also gives us a bit more flexibility if in some crazy universe in the future the properties and attributes don't match anymore.

Reply:
Hm, OK. I"m not a big fan of the getattr processing here. But I also don't have an idea at the moment how to avoid it if we want to try and load and copy optional things. So OK to keep as-is for now.